### ISSUE TITLE
Sass::SyntaxError in SabisuRails::Explorer#index

### ISSUE THREAD

**Issue by nfpiche** [IcaliaLabs/sabisu-rails#34]
Running into this issue trying to set up sabisu, somewhere in the @import 'furrato', the site works (although very ugly) if I comment out that line

Undefined mixin 'display-flex'.
  (in /home/nate/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/bundler/gems/sabisu-rails-af7499e87912/app/assets/stylesheets/sabisu_rails/application.css.scss:32)

**Comment by kurenn** [comment]
Yeah this is due to compass renamed many methods, I'm adressing this to update even to the last version of Furatto.

**Comment by lucasff** [comment]
+1

**Comment by kurenn** [comment]
Have you tried updating the sabisu gem?

**Comment by tuanpembual** [comment]
i found the same problems.
I just change my Gemfile to:

gem 'sabisu_rails', github: "IcaliaLabs/sabisu-rails"
gem 'furatto', github: "IcaliaLabs/furatto-rails"
gem 'compass-rails'

**Comment by kurenn** [comment]
Did that solve the problem?
